src/main/java/fi/metatavu/dcfb/server/search/handlers/AbstractIndexableHandler.java

```diff
@@ -63,6 +65,27 @@ public void deleteIndexable(UUID id) {
    */
   protected abstract String getType();
 
+  /**
+   * Creates GeoPoint for a Location object
+   * 
+   * @param location location
+   * @return GeoPoint
+   */
+  protected GeoPoint createGeoPoint(Location location) {
+    if (location == null) {
+      return null;
+    }
+    
+    BigDecimal latitude = location.getLatitude();
+    BigDecimal longitude = location.getLongitude();
+    
+    if (longitude == null || longitude == null) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Correct one of the identical sub-expressions on both sides of operator "||" ([squid:S1764](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1764))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=21&issues=AWVnq7t2XKENQn1Shuhp&open=AWVnq7t2XKENQn1Shuhp)

[ISSUE_KEY]: AWVnq7t2XKENQn1Shuhp